### ISSUE TITLE
`serialize: {immutable: Immutable}` option

### ISSUE THREAD

**Issue by unindented** [zalmoxisus/remote-redux-devtools#78]
On `redux-devtools-extension`, there's the option to pass `serialize: {immutable: Immutable}`. That option is not documented on this repo... Is there a reason for that?

**Comment by tsaiDavid** [comment]
@unindented  - If I'm not mistaken, it's not supported. The standard `redux-devtools-extension` is using his `remotedev-serialize`

@zalmoxisus - What approach do you suggest we take - I could possibly help with a PR since I need to use this.

**Comment by zalmoxisus** [comment]
It should be supported, as being added [here](https://github.com/zalmoxisus/remotedev-utils/commit/0d8226bb786ce26891c4bb5f92c2c579a8e294fa). A pr to update the docs would be much appreciated.

**Comment by tsaiDavid** [comment]
@zalmoxisus Wow, thanks for the quick reply - thanks for pointing that out! Sounds good, I can work on that. Will I have to use `remotedev-utils/src/importState.js`at all then? Thanks for all your work 👍

**Comment by zalmoxisus** [comment]
We need just need to import [`getSeralizeParameter`](https://github.com/zalmoxisus/remotedev-utils/commit/0d8226bb786ce26891c4bb5f92c2c579a8e294fa#diff-1fdf421c05c1140f6d71444ea2b27638R107) in [here](https://github.com/zalmoxisus/remote-redux-devtools/blob/master/src/devTools.js#L6) and use like so:
```js
const serialize = getSeralizeParameter(config)
```
Then pass it to [`stringify`](https://github.com/zalmoxisus/remotedev-utils/blob/0d8226bb786ce26891c4bb5f92c2c579a8e294fa/src/index.js#L94) from `remotedev-utils`, which should be used instead of  `jsan`'s `stringify`.

Also we'll need to use [`importState`](https://github.com/zalmoxisus/redux-devtools-extension/blob/f0e2ec8321a65f01ed210757dcb36db3816530b5/src/app/api/importState.js#L9) instead of https://github.com/zalmoxisus/remote-redux-devtools/blob/master/src/devTools.js#L99-L101.

Basically, the long-term plan is to move all the common part from Redux DevTools Extension into `remotedev-utils` and reimplement it here.

**Comment by unindented** [comment]
@tsaiDavid any progress here? How can I help? I don't know enough about this project's internals to understand @zalmoxisus's steps. If somebody can detail them even more I'll try.
